### DISCUSSION TITLE
Are translation corrections for the Craft backend welcome as an issue?

### DISCUSSION THREAD

**Discussion by mprofitlich** [craftcms/cms#15785] [Q&A] [answered]
I found a (minor) spelling mistalke in the German translation of the backend. Are you interested in receiving corrections? As an issue or differently?

**Comment by thupsi** [comment]
I think you can do a PR with the fix.

**Comment by brandonkelly** [comment, marked as answer]
An issue is fine. Translations are managed on CrowdIn, so a PR actually isn’t super helpful in this case.

**Comment by thupsi** [comment]
Ah, sorry for the wrong advice then @mprofitlich. I seem to misremember that it doesn't matter if you do it via CrowdIn or a PR. I did some greek translations in CrowdIn some time ago, but I actually started recently to prepare a large PR for Craft 5. Should I scrap that and do it in CrowdIn then @brandonkelly?

**Reply by brandonkelly** [reply]
@thupsi PR is fine if it’s a lot of changes – I can upload the new translation file directly to CrowdIn to be imported.
